### PR TITLE
Update db pool helm chart template

### PR DESCRIPTION
**Pull request by roguesherlock**

### CODE REVIEW

File: db-pool/values.yaml [hunk 1]
```diff
@@ -52,5 +52,7 @@ postgresql:
 # Mysql settings
 mysql:
```
Comment (iffyuva) [inline db-pool/values.yaml:53]: Remember to remove mysql dependency later.

**Review by iffyuva — CHANGES_REQUESTED** [review]
There should be no new line in `dot-env-file` :| Otherwise PR looks good.

**Comment by roguesherlock** [comment]
updated.

**Comment by iffyuva** [comment]
Tested for BTT deployments.

**Review by iffyuva — APPROVED** [review]

**Pull request merged**